package.json

```diff
@@ -41,13 +41,13 @@
     "@rollup/plugin-commonjs": "^19.0.0",
     "@rollup/plugin-node-resolve": "^13.0.0",
     "@rollup/plugin-typescript": "^8.2.1",
-    "@storybook/addon-actions": "^6.4.14",
-    "@storybook/addon-knobs": "^6.4.0",
-    "@storybook/addon-links": "^6.4.14",
-    "@storybook/addon-options": "^5.3.21",
-    "@storybook/addon-storysource": "^6.4.14",
-    "@storybook/addons": "^6.4.14",
-    "@storybook/svelte": "^6.4.14",
+    "@storybook/addon-essentials": "^7.2.1",
+    "@storybook/addon-interactions": "^7.2.1",
+    "@storybook/addon-links": "^7.2.1",
+    "@storybook/blocks": "^7.2.1",
+    "@storybook/svelte": "^7.2.1",
+    "@storybook/svelte-webpack5": "^7.2.1",
+    "@storybook/testing-library": "^0.2.0",
```

Comment:
Using node 20.8.1, I also needed to add:
`"@storybook/theming": "^7.4.6",` 

otherwise got an error running `npm start` after `npm install`